change/@ni-nimble-components-f4992b1c-b7f0-46e1-a684-b5afff8683f6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you comment why you went with only manual testing rather than adding auto test coverage for the new behavior? If there are technical challenges or precedent from similar components we should capture that in the PR description. Even better would be to add test coverage, for example via Chromatic snapshots, but if we're limited by the actual browser's size I could see that being challenging.

Reply:
I don't really have a good excuse. I think I felt like we already have pretty scattershot coverage of our components' dynamic behaviors, so I wasn't that compelled to go beyond manual testing. But we should, as there's no significant technical barrier that I'm aware of. Based on your response to my comment (about testing and behavior) in the other thread, I'll add the appropriate test cases.